mystyle.css

```diff
@@ -625,6 +677,34 @@ For small, less significant text and placeholders.
     flex-direction: column;
     align-items: center;
   }
+
+  .nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 70px;
+    gap: 0;
+    flex-direction: column;
+    background-color: yellow;
+    width: 100%;
+    text-align: center;
+    transition: 0.3s;
+  }
+
+  .navlink {
+    position: unset;
+    height: auto;
+    display: flex;
+    width: auto;
+    background-color: transparent;
+  }
```

Comment:
- [ ] One of the objectives of this project is to understand how to parse a Figma design precisely. Kindly update the `font-size` and position of the nav links. 
- [ ] When the nav menu is opened, it should fill the entire width of the screen. Please update it according to the Figma design. 🧐️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/210536256-c95f2738-998d-4356-9759-bfe2d27f0663.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/210536261-57b8ff34-5e02-4fa2-a8d8-4a6bac1dced5.png" width="300">|